handler/cantabular_metadata_export.go

```diff
@@ -204,40 +217,42 @@ func (h *CantabularMetadataExport) generateDownloadURL(e *event.CantabularMetada
 }
 
 // generateVaultPathForFile generates the vault path for the provided root and filename
-func (h *CantabularMetadataExport) generateVaultPath(instanceID string) string {
-	return fmt.Sprintf("%s/%s.txt", h.cfg.VaultPath, instanceID)
+func (h *CantabularMetadataExport) generateVaultPath(datasetID string) string {
+	return fmt.Sprintf("%s/%s.txt", h.cfg.VaultPath, datasetID)
```

Comment:
I don't know if this will be unique? We may need to follow the same format as the filename:
```
{vaultPath}/{datasetID}-{edition}-{version}
```
But maybe I'm wrong, we can merge and we might revisit this in the future if we have problems.